boards/nrf52840dongle/Makefile.dep

```diff
@@ -0,0 +1,13 @@
+ifneq (,$(filter saul_default,$(USEMODULE)))
+  USEMODULE += saul_gpio
+endif
+
+FEATURES_REQUIRED += bootloader_nrfutil
```

Comment:
Please move this line as suggested below. The `bootloader_nrfutil` is a convient build system feature that is only useful when stdio_cdc_acm is used !!

Reply:
Good point, fixed.

Reply:
Argh, sorry, I mixed up your suggestion with the one of benpicco "below" in the issue view. Does 844a0db625bbef185c06f9d47eca9863e0c3df19 (modulo points of opinion in the comment) reflect what you meant?